pkg/kubectl/cmd/get/get.go

```diff
@@ -665,8 +673,8 @@ func (o *GetOptions) watch(f cmdutil.Factory, cmd *cobra.Command, args []string)
 			objsToPrint = append(objsToPrint, obj)
 		}
 		for _, objToPrint := range objsToPrint {
-			if o.IsHumanReadablePrinter {
-				// printing always takes the internal version, but the watch event uses externals
+			if o.IsHumanReadablePrinter && objToPrint.GetObjectKind().GroupVersionKind().GroupKind() != tableGK {
+				// printing anything other than tables always takes the internal version, but the watch event uses externals
```

Comment:
reminds me for v1 we'll need two versions of this, beta1 and v1

Reply:
this one in particular just checks group/kind, so it's version neutral... places where we check for *metav1beta1.Table will definitely need to add a switch